docs/en/advanced_guides/add_dataset.md

```diff
@@ -58,99 +99,264 @@ We use `RepeatDataset` as wrapper to repeat the dataset.
 For example, suppose the original dataset is `Dataset_A`, to repeat it, the config looks like the following
 
 ```python
-dataset_A_train = dict(
+dataset_A_train_dataloader = dict(
```

Comment:
there is no necessary to config a `dataset_A_train_dataloader`, please remove it  and just keep dataset configuration